mica/archive/asp_l1.py

```diff
@@ -210,13 +210,13 @@ def get_atts_from_files(asol_files, acal_files, aqual_files, filter=True):
                        & (asol['time'] <= (aqual['time'][idx] + 1.025)))
                 asol = asol[~nok]
         # Make a Nx4 list of the inv misalign quats
-        q_mis_inv = np.repeat(Quat(acal['aca_misalign'][0]).inv().q,
+        q_mis_inv = np.repeat(Quat(transform=acal['aca_misalign'][0]).inv().q,
```

Comment:
It's probably worth writing this out a bit, this expression is really difficult for me to parse.
```
# Transpose of transform/rotation matrix is the inverse
aca_mis_inv = acal['aca_misalign'][0].transpose()
# Repeat the transform to match asol
aca_mis_inv = np.repeat(aca_mis_inv[np.newaxis, ...], repeats=len(asol), axis=0)  
q_mis_inv = Quat(transform=aca_mis_inv)
```

Reply:
Updated to be written out per Tom's comment.